runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java

```diff
@@ -57,10 +58,17 @@ public Dimension render(Graphics2D graphics)
 
 		for (NPC imp : imps)
 		{
-			Point minimapLocation = imp.getMinimapLocation();
-			if (minimapLocation != null)
+			Point impLocation = imp.getMinimapLocation();
+			if (impLocation != null)
 			{
-				OverlayUtil.renderMinimapLocation(graphics, minimapLocation, plugin.getIds().get(imp.getId()));
+				Color color = plugin.getIds().get(imp.getId());
+				OverlayUtil.renderMinimapLocation(graphics, impLocation, color);
+
+				if (plugin.getConfig().showName())
```

Comment:
You should inject the config into the constructor of the overlay, and then use that. This does not compile.